src/components/build/mainPage/mainPage.scss

```diff
@@ -85,22 +85,6 @@
     padding-top: 14.5vh;
   }
 
-  .zoom-item {
-    cursor: pointer;
-    text-align: center;
-    transition: all 0.2s ease-in-out;
-    transform-origin: 50% 75%;
-  }
-
-  .zoom-item:hover {
-    transform: scale(1.4);
-
-    .item-description {
-      color: white;
-      font-family: $font-family-bold;
-    }
-  }
-
```

Comment:
@farmanahmed2007 Why did you move this class to global.scss if it is used only in main page?

Reply:
I can use somewhere else and generalize the scale effect